src/actinia_parallel_plugin/api/job.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Copyright (c) 2018-2022 mundialis GmbH & Co. KG
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Endpoint definitions for job
+"""
+
+__license__ = "GPLv3"
+__author__ = "Carmen Tawalika, Anika Weinmann"
+__copyright__ = "Copyright 2018-2022 mundialis GmbH & Co. KG"
+__maintainer__ = "mundialis GmbH % Co. KG"
+
+from flask_restful import Resource
+from flask_restful_swagger_2 import swagger
+from flask import make_response, jsonify
+
+from actinia_core.models.response_models import \
+    SimpleResponseModel
+
+from actinia_parallel_plugin.resources.logging import log
+from actinia_parallel_plugin.core.jobs import (
+    getJob,
+)
+from actinia_parallel_plugin.apidocs import jobs
+
+
+class JobId(Resource):
+    """ Definition for endpoint standortsicherung
+    @app.route('/processes/standortsicherung/jobs/<jobid>')
```

Comment:
```suggestion
    """ Definition for endpoint standortsicherung
    @app.route('/processes/standortsicherung/jobs/<jobid>')
```

I assume that "standortsicherung" isn't an ideal word in this class...